src/vue-ctk-date-time-picker/vue-ctk-date-time-picker.vue

```diff
@@ -122,6 +122,7 @@
       formatted: { type: String, default: 'llll' },
       format: { type: String, default: String },
       locale: { type: String, default: 'en' },
+      timeZone: { type: String, default: 'America/Santiago' },
```

Comment:
For the default TZ, can you use : `moment.tz.guess()`

Like that, the user changes the TZ only if he needs